web/src/components/feature/FluidDialog.tsx

```diff
@@ -1,28 +1,55 @@
+import styled from 'styled-components'
 import { Button, Dialog, TextField } from '@equinor/eds-core-react'
+import { ComponentSelector } from './ComponentSelector'
+import { ComponentResponse } from '../../api/generated'
+
+const WideDialog = styled(Dialog)`
+  min-width: 700px;
+`
+
+const FluidPackageForm = styled.div`
+  display: flex;
+  flex-flow: row wrap;
+  gap: 30px;
+`
+
+const FirstColumn = styled.div`
+  display: flex;
+  flex-flow: column nowrap;
+  gap: 30px;
+`
```

Comment:
The styling is not quite right, I think. When selecting "2-2-dimethyl-butane(neohexane) (22-dm-C4)" (the longest component => ComponentSelector becomes wider), the selector jumps beneath the text fields.